modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -902,11 +936,20 @@
       }
     }
   },
-  "basePath": "/services/benefits/v2",
+  "basePath": "/services/claims/v2",
```

Comment:
We sure this is correct?
The Lighthouse platform is configured in DEV to hit our V2 endpoints at `/services/benefits/v2/`.
[REF](https://github.com/department-of-veterans-affairs/lighthouse-platform-tools/blob/main/environments/dev/config/vets-api.yaml#L267)

(fwiw, it's been a while since I worked in V2, but I have old Postman configs that also are hitting the base path as `/services/benefits/v2/`)

Reply:
Oh I just re-read the Jira ticket and the first bullet of required changes says that `basePath` needs to be removed.  So I'm guessing this attribute shouldn't exist here at all anymore?